index.html

```diff
@@ -2,12 +2,189 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-
     <link href="styles.css" rel="stylesheet" />
-
-    <title>Rocket Academy</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/brands.min.css"
```

Comment:
i think `brands` might be part of `all`? 🤔 

so there might not be a need to load both. that might bloat the load time

Reply:
maybe you can try to not load the `brands` file and see if the icons load as intended 😅